contents/articles/2017-08_pupil_mobile/index.md

```diff
@@ -0,0 +1,23 @@
+--- 
+title: Introducing Pupil Mobile
+date: Thu Aug 05 2017 14:36:00 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Pupil Mobile is a companion app to Pupil Capture and Pupil Service. The application enables you to connect your Pupil eye tracking headset to your Android device via USBC. Preview video and other sensor streams and stream video data over a WiFi..."
+featured_img: "../../../../media/images/blog/pupil_mobile.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/pupil_mobile.jpg"
+---
+
+We are excited to introduce Pupil Mobile! A companion app to Pupil Capture and Pupil Service, currently in public alpha at [Google Play Store](https://play.google.com/store/apps/details?id=com.pupillabs.pupilmobile)! 
```

Comment:
@willpatera - Do you know what version v0.9.x of Capture since it supported the app?